### ISSUE TITLE
500 | Server error on docker image

### ISSUE THREAD

**Issue by PeterNotenboom** [vitodeploy/vito#232]
**Describe the bug**
I receive a 500 | Server error when I visit the first launch

**To Reproduce**
Install via Docker:
`docker run -v vito_storage:/var/www/html/storage \
    -e APP_KEY=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx  \
    -e NAME=peter \
    -e EMAIL=myemail@here.com \
    -e PASSWORD=xxxxxxxxxxxxxxxxxx \
    -p 80:80 vitodeploy/vito:latest`

The docker container seems to run fine. I logged into the container itself and checked /var/log/nginx/error_log, but thats empty, while the access_log gives:
```
172.17.0.1 - - [07/Jun/2024:20:57:11 +0000] "GET / HTTP/1.1" 500 6622 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36 OPR/109.0.0.0"
```

**Expected behavior**
A launch page :)

**Screenshots**

![image](https://github.com/vitodeploy/vito/assets/8448364/3fb6a9ee-a299-44f6-9bed-42ce54875fec)

![image](https://github.com/vitodeploy/vito/assets/8448364/278c40de-3478-4c1f-ac51-070358556a48)

**Comment by PeterNotenboom** [comment]
Seems this is an easy fix. I tried running this on my NAS, now on my PC. That didn't solve it.
Solved it while creating this. (Rubby ducky?)

If your APP_KEY is longer than 32 characters, it wont give any errors or warnings. It will just give the 500 error above.
I accidently created a 33 character APP_KEY and that broke it.

Possible fix:
 - Give error and stop if the APP_KEY is longer/shorter than 32 characters
